configure.ac

```diff
@@ -20,7 +20,7 @@ License along with this program.  If not, see
 
 ]])
 
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
```

Comment:
Just curious, why going back to `2.69`? I believe I did the upgrade, but I honestly don't really know (or recall) the difference.

Reply:
Because 2.69 is sufficient and it imposes fewer restrictions on anyone compiling fibers.
(AFAIK autoconf's behaviour solely depends on the version of autoconf and doesn't change the AC_PREREQ in account).

More personally, while Guix does ship newer versions too, Guix uses version of 2.69 of autoconf by default, so it's convenient for me to lower the requirement a little.